modules/mcuboot/CMakeLists.txt

```diff
@@ -16,15 +16,16 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
     # SIGNED_HEX_FILE_OUT - (required) Signed hex output image
     # DEPENDS - (optional) One or more dependencies for signing the hex image
 
-    set(oneValueArgs SIGNED_BIN_FILE_IN SIGNED_HEX_FILE_NAME_PREFIX START_ADDRESS_OFFSET SIGNED_HEX_FILE_OUT)
+    set(oneValueArgs SIGNED_BIN_FILE_IN SIGNED_HEX_FILE_NAME_PREFIX SLOT_SIZE START_ADDRESS_OFFSET SIGNED_HEX_FILE_OUT)
```

Comment:
SLOT_SIZE not documented.

And why not just size, is it important for the sign command to know that it's a slot size ?
I would assume it just signs according to the size given.

Reply:
The sign command needs to know the slot size unfortunately as it puts some data at the end of the slot.
So the size of the image can be smaller than the slot size, but the sign command needs to know where it should offset the TLV data at the end of the image.

Reply:
Then `SLOT_SIZE` does sound like the best name.
But it is still not documented, please add description of SLOT_SIZE here:
https://github.com/nrfconnect/sdk-nrf/blob/4af2c283eea8e2a1d5f7663a0ebe659e2cbd7d82/modules/mcuboot/CMakeLists.txt#L12-L17

Please provide the same details as in your last comment, so readers understand the difference between image size and slot size.